s3_file_field/_multipart_boto3.py

```diff
@@ -16,13 +16,19 @@ def __init__(self, storage: 'S3Boto3Storage'):
         self._client: s3.Client = resource.meta.client
         self._bucket_name: str = storage.bucket_name
 
-    def _create_upload_id(self, object_key: str) -> str:
+    def _create_upload_id(
+        self,
+        object_key: str,
+        content_type: Union[str, None] = None,
+    ) -> str:
+        boto3_kwargs = {}
+        if content_type is not None:
+            boto3_kwargs['ContentType'] = content_type
         resp = self._client.create_multipart_upload(
             Bucket=self._bucket_name,
             Key=object_key,
+            **boto3_kwargs,  # type: ignore
```

Comment:
What's the MyPy error here? If it can't be fixed, could you use a more specific ignore?

Reply:
I have an idea for how to fix this. I'll try locally.